ui/src/app/utils/research-purpose.ts

```diff
@@ -0,0 +1,26 @@
+import {ResearchPurposeItems, SpecificPopulationItems} from 'app/pages/workspace/workspace-edit';
+import {SpecificPopulationEnum, Workspace} from 'generated/fetch';
+
+export function getSelectedResearchPurposeItems(workspace: Workspace) {
```

Comment:
hooray for reuse!

Would it make sense to have the parameters be `ResearchPurpose` instead, since that is all these functions need?